version.txt

```diff
@@ -0,0 +1 @@
+0.6.1-alpha.0
```

Comment:
You'll just need to modify this for the release-0.5 branch

Reply:
good point. I have updated. thanks